config/jobs/kubernetes/sig-node/README.md

```diff
@@ -1,169 +1,205 @@
-## Signode CI Test Coverage Guidance
+# SIG Node CI Test Coverage Guidance
 
+## Test Tabs
 
-## Variables
+#### 1. **sig-node-release-informing**
 
+This test tab should be checked in the weeks prior to code freeze and release,
+and signals general stability. 
 
 <table>
   <tr>
-   <td><strong>Variables</strong>
+   <td>
+<strong>Variables</strong>
    </td>
    <td><strong>Values</strong>
    </td>
   </tr>
   <tr>
    <td>cri
    </td>
-   <td>Containerd, CRI-O
+   <td>containerd,  CRI-O
    </td>
   </tr>
-  <tr>
    <td>os
    </td>
-   <td>cos, ubuntu, fedora
+   <td>COS, ubuntu
    </td>
   </tr>
   <tr>
-   <td>release branch
+   <td>release-branch
    </td>
-   <td>N-3, N-2, N-1, N, main
+   <td>master
+   </td>
+  </tr>
+  <tr>
+   <td>test-type
+   </td>
+   <td>node-conformance, serial, standalone, standalone-serial, flaky
    </td>
   </tr>
 </table>
 
+**Test Names**
 
+(pull|ci)-node-{cri}-{os}-{release-branch}-{test-type}
 
-## Naming Convention
+For node-conformance, serial, and standalone tests will also have the following jobs:
+* (pull|ci)-node-{cri}-{os}-{release-branch}-{test-type}-beta-enabled
+  * All beta features will be **enabled**, but corresponding beta tests will not run.
+* (pull|ci)-node-{cri}-{os}-{release-branch}-{test-type}-alpha-and-beta-enabled
+  * All alpha and beta features will be **enabled**, but corresponding alpha and beta tests will not run.
 
-(pull|ci)-{$cri}-{$os}-{$test-type}-release-{$release-branch}
+**Test Args**
 
-**Note:** Specify variable value in the name only if it isn’t set to default in a test tab.
+* node-conformance
+  * Focus: `NodeConformance`
+  * Skip: `Flaky`, `Slow`, `Serial`
+* serial
+  * Focus: `Serial`
+  * Skip: `Flaky`, `Benchmark`, `Feature:.+`, `FeatureGate:.+`
+* standalone
+  * Focus: `Feature:StandaloneMode`
+  * Skip: `Flaky`, `Serial`, `Feature:.+`, `FeatureGate:.+`
+* standalone-serial
+  * Focus: `Feature:StandaloneMode`, `Serial`
+  * Skip: `Flaky`, `Feature:.+`, `FeatureGate:.+`
+* flaky
+  * Focus: `Flaky`
+  * Skip: `Benchmark`, `Feature:.+`, `FeatureGate:.+`
 
+#### 2. **sig-node-node-conformance**
 
-## Test Tabs
+This tab is is focused on node conformance tests. 
 
-#### 1. **sig-node-default (i.e. rename sig-node-release-blocking)**
+**Test Names**
+
+This tab will include all of the **node-conformance** tests from the previously described **sig-node-release-informing** tab,
+and will additionally have the following jobs:
 
 <table>
   <tr>
    <td>
 <strong>Variables</strong>
    </td>
-   <td><strong>Default Values</strong>
+   <td><strong>Values</strong>
    </td>
   </tr>
   <tr>
    <td>cri
    </td>
-   <td>containerd,  CRI-O
+   <td>containerd
```

Comment:
I know we sometimes struggle to keep them green but I'd love cri-o tests to be kept here if possible